docs/pages/prometheus.md

```diff
@@ -4,4 +4,93 @@ title: Monitoring with Prometheus
 categories: ['User Manual']
 order: 7.5
 ---
-Coming soon.
+Run Scanner was built to be monitored by [Prometheus](https://prometheus.io/).
+Add Run Scanner as a target of your Prometheus server.
+
+Most variables exported by Run Scanner are prefixed with `miso_runscanner_`.
+The default JVM metrics are exported with the prefix `jvm_`.
+
+As a default, we suggest the following alerts:
+
+```
+groups:
+- name: runscanner.rules
+  rules:
+  - alert: StuckRuns
+    expr: sum_over_time(miso_runscanner_new_runs_scanned[1h]) - sum by(instance, environment,
+      job) (rate(miso_runscanner_waiting_runs[1h])) < 3 and time() - process_start_time_seconds  > 8 * 3600
+    annotations:
+      description: The runs being processed by {{$labels.instance}} are not being
+        processed and cleared.
+      summary: Runscanner {{$labels.instance}} seems to be stuck
+  - alert: BadRuns
+    expr: miso_runscanner_directories_attempted - miso_runscanner_directories_accepted
+      > 0
+    annotations:
+      description: The run scanner {{$labels.instance}} has found candidate sequencer
+        output directories that it does not have permission to read.
+      summary: Unreadable directories run directories on {{$labels.instance}}
+  - alert: AutoInhibit
+    expr: time() - process_start_time_seconds{job="runscanner"}  < 15 * 60 or sum(miso_runscanner_waiting_runs) by (environment, instance, job) - miso_runscanner_bad_runs > 5
+    labels:
+      scope: runscanner
+    annotations:
+      description: Run Scanner was restarted recently and probably needs time to finish scanning old runs
+      summary: Runscanner cache is cold
+```
+
+`StuckRuns` will fire when Run Scanner no longer seems to be making progress
+extracting data from run directories. This usually occurs if I/O latency has
+increased or CPU starvation.
```

Comment:
"This usually occurs if [...] CPU starvation" -> "This usually occurs if I/O latency has increased or in the case of CPU starvation"?